google/cloud/forseti/scanner/scanners/config_validator_util/cv_data_converter.py

```diff
@@ -59,6 +59,27 @@ def generate_ancestry_path(full_name):
     return ancestry_path
 
 
+def convert_data_to_cai_asset(primary_key, resource, resource_type):
+    """Convert data to CAI formatted fields to be used by Config Validator.
+
+    Args:
+        primary_key (str): The unique identifier of the resource object.
+        resource (json): The resource object that will be converted.
+        resource_type (str): The resource type (e.g. lien, sink, etc.).
+
+    Returns:
+        json: converted resource object.
```

Comment:
This would be more clear to future reader with something like this: `json: converted resource object in CAI data structure.`

Reply:
Agreed- done!